embassy-rp/Cargo.toml

```diff
@@ -15,6 +15,7 @@ src_base_git = "https://github.com/embassy-rs/embassy/blob/$COMMIT/embassy-rp/sr
 features = ["defmt", "unstable-pac", "time-driver"]
 flavors = [
     { name = "rp2040", target = "thumbv6m-none-eabi" },
+    { name = "rp235x", target = "thumbv8m.main-none-eabi" },
```

Comment:
you have to enable the chip features for each flavor. Also i'd do one for the `a` and another for the `b` variant since it affects docs (more GPIO pins)